scipy/special/basic.py

```diff
@@ -754,20 +754,97 @@ def riccati_yn(n, x):
 
 
 def erfinv(y):
-    """Inverse function for erf.
+    """Inverse error function.
```

Comment:
`erf` should appear in this description on the fist line somewhere.  Perhaps
`Inverse of the error function erf.`